### ISSUE TITLE
Process gpx file with offset

### ISSUE THREAD

**Issue by SamR1** [SamR1/FitTrackee#156]
For now, FitTrackee only supports timezone-unaware gpx files.  
Some applications generate gpx files with offset, which may result in incorrect times.

**Comment by SamR1** [comment]
needs #35

**Comment by albertodiazsaez** [comment]
I just had a problem with this, I was going to report it as a new issue, but I think it fits here:

Yesterday I recorded a track, which started at 22:32 local time (CEST), which has this <time> tag in the GPX: 2022-03-27T22:32:01.137+02:00, but it gets marked as a 28 of March track (the track ended before midnight, so it's full in the 27 of March). I don't know if you are aware of this, because you only say "incorrect times" and not dates, so I will give you a GPX file which gives me this problem (modified to hide real coordinates, but it still fails to get the date right when uploading it).

[date_problem_file.zip](https://github.com/SamR1/FitTrackee/files/8362773/date_problem_file.zip)

I recorded it using OpenTracks with the ISO 8601 date type. From now on and until the issue is fixed, I will use "local" dates.
